templates/cloudflare-pages/package.json

```diff
@@ -8,7 +8,7 @@
     "dev": "remix build && run-p \"dev:*\"",
     "start": "cross-env NODE_ENV=production npm run wrangler",
     "typecheck": "tsc -b",
-    "wrangler": "wrangler pages dev ./public"
+    "wrangler": "wrangler pages dev ./public --compatibility-flags streams_enable_constructors"
```

Comment:
I think putting this in `wrangler.toml` would be a better idea, as that makes all config is in 1 place (just like @lili21 did in #4502)